Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
Maybe that could work. I'm going to be on vacation until May 19, so I won't
be able to work on it again until then.

Thanks for all the help!
On May 6, 2015 11:58 AM, "Dominyk Tiller" <notifications@github.com> wrote:

> In Library/Formula/wxmac.rb
> <https://github.com/Homebrew/homebrew/pull/37003#discussion_r29772424>:
>
> > @@ -67,8 +68,13 @@ def install
> >        "--disable-monolithic"
> >      ]
> >
> > -    system "./configure", *args
> > +    system "./configure", "--enable-shared", *args
>
> Thanks for the detailed explanation [image: :bow:]. Perhaps instead of
> redoing the whole make install process again you could just do make and
> then lib.install Dir["*.a"] or something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/37003/files#r29772424>.
>